### ISSUE TITLE
How do I install this mod?

### ISSUE THREAD

**Issue by SoullessWaffle** [Sollace/Presence-Footsteps#5]
I'm playing the [Minecraft 1.8.9 Pioneers](http://minecraft.curseforge.com/projects/the-pioneers) pack (installed through the [Curse launcher](https://www.curse.com/)) and I'd really like to add this mod as well as Matmos to it.

Adding mods to a pack usually just means dropping the `.jar` into the `/mods` folder, but because this mod seems to require liteloader and doesn't have a `.jar` file, I'm a bit confused.

Thanks in advance.

**Comment by Sollace** [comment]
You can go get liteloader and use the installer to get a .jar to put in the mods folder.

After that it should work to put the .litemod into /mod/1.8.9

**Comment by SoullessWaffle** [comment]
Thanks for your quick reply.

I put the liteloader `.jar` into the `/mods` folder, `presencefootsteps-u8d.litemod` into `/mods/1.8.9` and `PF.Resources.zip` into `/resourcepacks`, but now I'm getting a crash after clicking Play in the Minecraft launcher: http://pastebin.com/kZHP9VmX

The `liteloader-1.8.jar` is supposed to work with 1.8.9, right? It's the latest version I could find, I got it from [here](http://jenkins.liteloader.com/job/LiteLoaderInstaller%201.8/).

**Comment by Sollace** [comment]
There is a separate version for 1.8.9. Try this [one](http://jenkins.liteloader.com/view/1.8.9/job/LiteLoaderInstaller%201.8.9/).

**Comment by SoullessWaffle** [comment]
Thanks a lot! Using that instead of `liteloader-1.8.jar` worked.

I did have a (minor?) issue shortly after installing the mod, this is what I did to encounter it:

1. Launch Minecraft
2. Load a save
3. Pause the game
4. Go into the options menu and select the resource pack
5. Apply it
6. Resume the game

At this point I could already hear the mod working when I walked around, but I would get _massive_ freezing/slowdown about every second and the game was basically unplayable.

I'm not sure what caused it, or if it's even reproducible, but restarting the game fixed it and I'm no longer experiencing any lag/freezing.
